cg/server/invoices/templates/invoices/layout.html

```diff
@@ -46,7 +46,20 @@
         </div>
       </nav>
 
-    
+  {% with messages = get_flashed_messages() %}
+    {% if messages %}
+        <ul class=flashes>
+        {% for message in messages %}
+        <div class="alert alert-danger alert-dismissible fade show" role="alert">
+            {{ message }}
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+        {% endfor %}
+        </ul>
+      {% endif %}
```

Comment:
A `ul` cannot have a `div` as direct child: https://stackoverflow.com/questions/11755628/can-i-use-div-as-a-direct-child-of-ul

Can we use a `div` instead of a `ul`?